CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 All notable changes are documented in this file. Release numbers follow [Semantic Versioning](http://semver.org)
 
+## 0.7.4
+March 7th, 2018 [Detailed release notes](https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12317320&version=12336214)
+
+### New features since 0.7.3
+
+* Provide MySQL snapshot mode that does not require table locks [DBZ-602](https://issues.jboss.org/browse/DBZ-602)
+* Add support for columns of type "bytea" [DBZ-605](https://issues.jboss.org/browse/DBZ-605)
+* Add string as an option for decimal.handling.mode [DBZ-611](https://issues.jboss.org/browse/DBZ-611)
+* Support CREATE TABLE statements with PARTITION ... ENGINE=InnoDB [DBZ-641](https://issues.jboss.org/browse/DBZ-641)
+* Document VariableScaleDecimal in PG connector docs [DBZ-631](https://issues.jboss.org/browse/DBZ-631)
```

Comment:
I'd say this and all below don't really qualify as "new feature" but rather as "fix or change". No need to change it now, but let's do a quick pass on issue titles and categories before doing a release in general.